server/deadfish.hpp

```diff
@@ -71,7 +71,6 @@ struct Mob : public Collideable {
 
 struct Player : public Mob {
 	std::string name;
-	websocketpp::connection_hdl conn_hdl;
```

Comment:
Was moving the connection handle to the `Mob` class intentional?

Reply:
It was not. It's pretty incredible that it compiled and worked correctly even though this was a mistake :P